doc/source/Resources/pyaedt_with_IDE.bat

```diff
@@ -108,13 +109,26 @@ if [%install_pyaedt%]==[y] (
     )
     call "%pyaedt_install_dir%\Scripts\activate.bat"
     if NOT [%wheelpyaedt%]==[] (
-        echo Installing Pyaedt from local wheels %arg1%
+        echo Installing PyAEDT from local wheels %arg1%
```

Comment:
Punctuation please. Does the message string end with the word wheels or after the %arg1%? If you teach me, I'll know how to make the edit next time!